lib/hammer_cli_foreman/compute_resource/vmware.rb

```diff
@@ -1,68 +1,79 @@
 module HammerCLIForeman
   module ComputeResources
     class VMware < Base
-      INTERFACE_TYPES = %w(
-           VirtualVmxnet3,
-           VirtualE1000
-         )
+      INTERFACE_TYPES = %w[VirtualVmxnet3 VirtualE1000]
 
       def name
-        _('VMware')
+        'VMware'
       end
 
       def compute_attributes
         [
-            ['cpus',                 _('CPU count')],
-            ['corespersocket',       _('Number of cores per socket (applicable to hardware versions < 10 only)')],
-            ['memory_mb',            _('Integer number, amount of memory in MB')],
-            ['firmware',             'automatic/bios/efi'],
-            ['cluster',              _('Cluster ID from VMware')],
-            ['resource_pool',        _('Resource Pool ID from VMware')],
-            ['path',                 _('Path to folder')],
-            ['guest_id',             _('Guest OS ID form VMware')],
-            ['scsi_controller_type', _('ID of the controller from VMware')],
-            ['hardware_version',     _('Hardware version ID from VMware')],
-            ['add_cdrom',            _('Must be a 1 or 0, Add a CD-ROM drive to the virtual machine')],
-            ['cpuHotAddEnabled',     _('Must be a 1 or 0, lets you add memory resources while the machine is on')],
-            ['memoryHotAddEnabled',  _('Must be a 1 or 0, lets you add CPU resources while the machine is on')],
-            ['annotation',           _("Annotation Notes")]
+          ['cpus',                 _('CPU count')],
+          ['corespersocket',       _('Number of cores per socket (applicable to hardware versions < 10 only)')],
+          ['memory_mb',            _('Integer number, amount of memory in MB')],
+          ['firmware',             'automatic/bios/efi'],
+          ['cluster',              _('Cluster ID from VMware')],
+          ['resource_pool',        _('Resource Pool ID from VMware')],
+          ['path',                 _('Path to folder')],
+          ['guest_id',             _('Guest OS ID form VMware')],
+          ['hardware_version',     _('Hardware version ID from VMware')],
+          ['memoryHotAddEnabled',  _('Must be a 1 or 0, lets you add CPU resources while the machine is on')],
+          ['cpuHotAddEnabled',     _('Must be a 1 or 0, lets you add memory resources while the machine is on')],
+          ['add_cdrom',            _('Must be a 1 or 0, Add a CD-ROM drive to the virtual machine')],
+          ['start',                _('Must be a 1 or 0, whether to start the machine or not')],
+          ['annotation',           _('Annotation Notes')],
+          ['scsi_controllers',     _('List with SCSI controllers definitions')]
         ]
       end
 
-      def host_attributes
```

Comment:
start a VM is an action that is not part of the compute attributes (when creating a compute profile, that's why we had the host_attributes

Reply:
You're right, returned to the previous state.